### PR TITLE
BLADE-425 Don't fail to load if an embedded extension fails

### PR DESCRIPTION
**Pull request by christopherbryanboyd**

### CODE REVIEW

File: cli/src/main/java/com/liferay/blade/cli/ExtensionsClassLoaderSupplier.java [hunk 1]
```diff
@@ -137,15 +139,54 @@ private void _extractBladeExtensions(Path extensionsDirectory) throws IOExceptio
...
+					else {
+						String errorString = String.format("Unable to locate %s on classpath", extension);
 
-					Files.copy(extensionInputStream, extensionPath, StandardCopyOption.REPLACE_EXISTING);
+						throw new NoSuchElementException(errorString);
+					}
 				}
+				
+
+				for (String extension : extensions) {
```
Comment (gamerson) [inline cli/src/main/java/com/liferay/blade/cli/ExtensionsClassLoaderSupplier.java]: Lets have two try/catch blocks.  one that covers all this code (like you have here) and another try/catch for indiviidual extensions, incase there is a bad actor (uncopiable extension file) or something, but having two levels of try/catch would let us load more etensions if there is a bad one in the list.

**Comment by christopherbryanboyd** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by christopherbryanboyd** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by gamerson** [comment]
Pull request submitted to https://github.com/christopherbryanboyd/liferay-blade-cli/pull/26. [See changes here.](https://github.com/gamerson/liferay-blade-cli/compare/c4b0fe64~2...c4b0fe64) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**